src/controllers/confirm.company.controller.ts

```diff
@@ -4,12 +4,12 @@ import ObjectionCompanyProfile from "model/objection.company.profile";
 import { SESSION_OBJECTION_ID } from "../constants";
 import { OBJECTIONS_ENTER_INFORMATION, OBJECTIONS_NO_STRIKE_OFF, OBJECTIONS_NOTICE_EXPIRED } from "../model/page.urls";
 import { Templates } from "../model/template.paths";
-import { ApiError, ObjectionStatus } from "../modules/sdk/objections";
+import {ApiError, ObjectionCreate, ObjectionStatus} from "../modules/sdk/objections";
```

Comment:
we had a rule in tslint to have spaces before and after braces, looks like it might have gone missing from eslint or maybe eslint didn't run. We'd need that rule switched on and spaces around the braces.

Reply:
Ok been through and added spaces to the braces.